### ISSUE TITLE
Use mobile site of DM5 for image scraping

### ISSUE THREAD

**Issue by flamyoad** [flamyoad/Honnoki#6]
In the Desktop site, you have to go through `chapterfun.ashx` AJAX call which require the parameters `cid`, `mid`, `dt` and `sign` which are hard to get.

Example call would be like this:
`https://www.dm5.com/m1119302/chapterfun.ashx?cid=1119302&page=2&key=&language=1&gtk=6&_cid=11102&_mid=3719&_dt=2021-06-16+19%3A37%3A23&_sign=d4c1882dab089da1b8fee7df7f4010` 

which will return a piece of obsfucated JS function like below in `text/html`
```
eval(function(p,a,c,k,e,d){e=function(c){return(c<a?"":e(parseInt(c/a)))+((c=c%a)>35?String.fromCharCode(c+29):c.toString(36))};if(!''.replace(/^/,String)){while(c--)d[e(c)]=k[c]||e(c);k=[function(e){return d[e]}];e=function(){return'\\w+'};c=1;};while(c--)if(k[c])p=p.replace(new RegExp('\\b'+e(c)+'\\b','g'),k[c]);return p;}('n 5(){2 6=4;2 8=\'9\';2 7="o://l-m-p-s-t.q.r/c/e/4";2 3=["/b.1","/f.1","/j.1","/g.1","/h.1","/k.1","/a.1","/u.1","/E.1","/F.1","/D.1","/G.1","/J.1","/H.1","/I.1","/C.1","/x.1","/w.1","/v.1"];y(2 i=0;i<3.B;i++){3[i]=7+3[i]+\'?6=4&8=9&A=\'}z 3}2 d;d=5();',46,46,'|jpg|var|pvalue|1119302|dm5imagefun|cid|pix|key|3c589631a547362326e74f897e38c3fe|8_6617|2_5627|38||37519|3_9375|5_5843|6_6362||4_9653|7_9036|manhua1034|104|function|https|250|cdnmanhua|net|150|12|9_9018|20_9906|19_2435|18_3674|for|return|uk|length|17_1753|12_9202|10_5042|11_8440|13_5215|15_3822|16_9820|14_5370'.split('|'),0,{}))
```
By going to the mobile site, we don't have to go through the AJAX call which means there is no need to pass our verification details. The obsfucated JS is attached directly inside the HTML of the mobile site.

<img width="625" alt="Screenshot 2021-06-16 at 7 43 12 PM" src="https://user-images.githubusercontent.com/35066207/122213058-20e57380-cedb-11eb-94e0-c03240967a4a.png">

TLDR;
https://www.dm5.com/m1119302/   ---->  https://m.dm5.com/m1119302/
